docling_serve/app.py

```diff
@@ -85,9 +85,17 @@ async def lifespan(app: FastAPI):
 
 
 def create_app():
+    try:
+        version = importlib.metadata.version("docling_serve")
+    except importlib.metadata.PackageNotFoundError:
```

Comment:
This shoud be possible if docling_core isn't installed as package for some reason